axum-core/src/response/into_response.rs

```diff
@@ -0,0 +1,507 @@
+use super::{IntoResponseParts, Response, ResponseParts};
+use crate::{body, BoxError};
+use bytes::{buf::Chain, Buf, Bytes, BytesMut};
+use http::{
+    header::{self, HeaderMap, HeaderValue},
+    StatusCode, Version,
+};
+use http_body::{
+    combinators::{MapData, MapErr},
+    Empty, Full, SizeHint,
+};
+use std::{
+    borrow::Cow,
+    convert::Infallible,
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+/// Trait for generating responses.
+///
+/// Types that implement `IntoResponse` can be returned from handlers.
+///
+/// # Implementing `IntoResponse`
+///
+/// You generally shouldn't have to implement `IntoResponse` manually, as axum
+/// provides implementations for many common types.
+///
+/// However it might be necessary if you have a custom error type that you want
+/// to return from handlers:
+///
+/// ```rust
+/// use axum::{
+///     Router,
+///     body::{self, Bytes},
+///     routing::get,
+///     http::StatusCode,
+///     response::{IntoResponse, Response},
+/// };
+///
+/// enum MyError {
+///     SomethingWentWrong,
+///     SomethingElseWentWrong,
+/// }
+///
+/// impl IntoResponse for MyError {
+///     fn into_response(self) -> Response {
+///         let body = match self {
+///             MyError::SomethingWentWrong => "something went wrong",
+///             MyError::SomethingElseWentWrong => "something else went wrong",
+///         };
+///
+///         // its often easier to implement `IntoResponse` by calling other implementations
```

Comment:
```suggestion
///         // its often easiest to implement `IntoResponse` by calling other implementations
```

… since there is nothing "easier" relates to.